Source/carthage/Archive.swift

```diff
@@ -25,7 +25,15 @@ public struct ArchiveCommand: CommandType {
 					|> flatMap(.Concat) { platform in
 						let framework = platform.relativePath.stringByAppendingPathComponent(options.frameworkName).stringByAppendingPathExtension("framework")!
 						let dSYM = framework.stringByAppendingPathExtension("dSYM")!
-						return SignalProducer(values: [framework, dSYM])
+						if NSFileManager.defaultManager().fileExistsAtPath(framework) {
```

Comment:
I'm refactoring this a little, but I'm going to keep the file check here, because I don't like the idea of suppressing other errors arising from the `dwarfdump`.